### ISSUE TITLE
Cannot detach OVS system interface from OVS bridge

### ISSUE THREAD

**Issue by cathay4t** [nmstate/nmstate#809]
Due to NetworkMaanager bug https://bugzilla.redhat.com/show_bug.cgi?id=1804167 we cannot
detach system interface from OVS bridge.

Test case `test_create_and_remove_ovs_bridge_with_a_system_port`  will show the
system interface as down state after removing the OVS bridge. It could be related to this
issue as interface cannot detach from OVS bridge.

**Comment by cathay4t** [comment]
To test this issue, apply these two yml files sequentially:

```yaml
---
interfaces:
- name: eth1
  type: ethernet
  state: up
- name: br0
  type: ovs-bridge
  state: up
  bridge:
    port:
    - name: ovs0
    - name: eth1
- name: ovs0
  type: ovs-interface
  state: up
```

Then

```yaml
---
interfaces:
- name: br0
  type: ovs-bridge
  state: up
  bridge:
    port:
    - name: ovs0
```

**Comment by cathay4t** [comment]
This is resolved by NM in NetworkManager-1.22.8

**Comment by cathay4t** [comment]
This issue is only fixed after NM 1.22.8 released. So currently our CI still fail on this test case
